analyzer/src/funTest/kotlin/managers/ConanFunTest.kt

```diff
@@ -71,12 +72,48 @@ class ConanFunTest : StringSpec() {
                 url = normalizeVcsUrl(vcsUrlPy)
             )
 
-            val result = createConan().resolveSingleProject(definitionFile)
+            val result = createConanDynamicVersions().resolveSingleProject(definitionFile)
+
+            patchActualResult(result.toYaml()) shouldBe expectedResult
+        }
+
+        /**
+         * This test cannot complete successfully when you run it locally as `package id` differs depending on operating
+         * system. The `package id` is set to the one calculated on Linux.
+         */
+        "Project dependencies are detected correctly with the lockfile" {
```

Comment:
Please configure the test so that it only runs on Linux:
```
"Project dependencies are detected correctly with the lockfile".config(enabled = Os.isLinux) {
```
(using `org.ossreviewtoolkit.utils.common.Os`)